source/unified-test-format/unified-test-format.rst

```diff
@@ -1885,10 +1913,12 @@ loop
 ~~~~
 
 The ``loop`` operation executes sub-operations in a loop.
-It supports the following arguments:
 
-- ``operations``: the sub-operations to run on each loop iteration.
-  Each sub-operation must be a valid operation as described in
+The following arguments are supported:
+
+- ``operations``: Required array of `operation`_ objects. List of operations
```

Comment:
While clarifying this, I realized that `tests[].operations` currently states that it requires "one or more" operations. That was actually never codified in the schema, and I actually relied on that fact when implementing various "valid-fail" tests. For tests that are unrelated to operations (e.g. entity options), I frequently use an empty array for `tests[].operations`.

`withTransaction` and `loop` never mentioned that "one or more" operations were required, so I propose they remain as-is and allow an empty array. 

I've opened [DRIVERS-1593](https://jira.mongodb.org/browse/DRIVERS-1593) to track this.